src/Sql/Sql/Database Backup/Cmdlet/SetAzureRmSqlDatabaseBackupShortTermRetentionPolicy.cs

```diff
@@ -64,7 +69,7 @@ protected override IEnumerable<AzureSqlDatabaseBackupShortTermRetentionPolicyMod
                     ResourceGroupName,
                     ServerName,
                     DatabaseName,
-                    new Management.Sql.Models.BackupShortTermRetentionPolicy(retentionDays: RetentionDays))
+                    new Management.Sql.Models.BackupShortTermRetentionPolicy(retentionDays: RetentionDays, diffBackupIntervalInHours: DiffBackupIntervalInHours))
```

Comment:
(2/2) convert `int` to `int?` then pass it to management SDK
```suggestion
                    new Management.Sql.Models.BackupShortTermRetentionPolicy(retentionDays: RetentionDays, diffBackupIntervalInHours: this.IsParameterBound(c => c.DiffBackupIntervalInHours) ? DiffBackupIntervalInHours : null as int?))
```

Reply:
Modified. For parameter retentionDays and diffBackupInterval: 
1. When they are both null -> throw "Parameter 'RetentionDays' and 'DiffBackupIntervalInHours' both being null is not allowed." error -> we expect this -> done. 
However, 
2. When one parameter is null and the other one has value -> it also throw " Nullable object must have a value." error -> we don't expect this since we want the configured parameter can go to server and been updated. The server can ignore the other parameter.

Reply:
I modified as I guessed and it works perfect now. Either retentionDays or diffBackupInterval can be set individually.